app/src/main/res/layout/activity_splash.xml

```diff
@@ -35,6 +35,7 @@
         android:background="@android:color/transparent"
         android:drawableStart="@drawable/create_contest"
         android:text="@string/create_contest"
+        android:textColor="@color/contestAppTextColor"
```

Comment:
typically its better to do it right away, but I'll allow it to prevent you guys from doubling up on work.